src/main/java/seedu/address/logic/commands/MarkPersonDescriptor.java

```diff
@@ -159,4 +159,24 @@ public void setTags(Set<Tag> tags) {
     public Optional<Set<Tag>> getTags() {
         return (tags != null) ? Optional.of(Collections.unmodifiableSet(tags)) : Optional.empty();
     }
+
+    @Override
+    public boolean equals(Object other) {
+        // for same object
+        if (other == this) {
+            return true;
+        }
+
+        // checks for null object
+        if (!(other instanceof MarkPersonDescriptor)) {
+            return false;
+        }
+
+        // check for same values in fields
```

Comment:
would be good to remove the other unnecessary fields from the class

Reply:
Can can, have made the changes